src/app/components/developer/developer-view.jsx

```diff
@@ -24,9 +24,9 @@ import { UserContext } from 'shared/contexts';
 const useStyles = makeStyles({
   content: {
     display: 'grid',
-    gridTemplateColumns: '1fr 1fr',
-    minHeight: '286px',
-    gap: '8px',
+    gridTemplateColumns:'1fr 1fr',
+    gap: '16px',
+    justifyItems:'stretch'
```

Comment:
There should be a space after the `:`. Also, is this required? I haven't seen what this does, but am curious